src/WebJobs.Script.Grpc/Channel/GrpcWorkerChannel.cs

```diff
@@ -718,7 +718,7 @@ internal async Task SendInvocationRequest(ScriptInvocationContext context)
                 // do not send an invocation request if cancellation has been requested
                 if (context.CancellationToken.IsCancellationRequested)
                 {
-                    _workerChannelLogger.LogWarning("Cancellation has been requested. The invocation request with id '{invocationId}' is cancelled and will not be sent to the worker.", invocationId);
+                    _workerChannelLogger.LogWarning("Cancellation has been requested. The invocation request with id '{invocationId}' is canceled and will not be sent to the worker.", invocationId);
```

Comment:
:eagle:

Reply:
🇬🇧☠️😢

Reply:
We do standardize on `en-US` in the codebase, so we should have this reverted :)

We also want to avoid log verbiage changes (unless absolutely necessary) as those could be breaking.

Reply:
`canceled` is US and `cancelled` is UK, so I was updating to follow the en-US standard

Reply:
I completely flipped your update and somehow saw this going the other way around 🤦🏼 

The concern about the verbiage change is still valid, but I agree with you that those are unlikely to cause issues. 

I'm good with the change.